*** tests/A-10-03-M-ok.out	2024-10-18 19:27:35.653007400 +0100
--- tests/A-10-03-M-ok.outhyp	2024-10-28 22:59:04.741899500 +0000
***************
*** 49,54 ****
--- 49,55 ----
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
+ 5 - Ver empregado com mais responsabilidades que X
  0 - Sair
  Escolha uma opção: Identificador único do habitat: ANIMAL|A1|Pinguim 1|C1|VOID|AR1
  ANIMAL|A3|Macaco 1|C2|VOID|AR1
***************
*** 58,63 ****
--- 59,65 ----
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
+ 5 - Ver empregado com mais responsabilidades que X
  0 - Sair
  Escolha uma opção: Identificador único do habitat: ANIMAL|A2|Pinguim 2|C1|VOID|AR2
  Consultas
***************
*** 65,70 ****
--- 67,73 ----
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
+ 5 - Ver empregado com mais responsabilidades que X
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Criar ficheiro
